mysql-test/t/mysql_57_inplace_upgrade.test

```diff
@@ -358,6 +358,28 @@ SELECT * FROM test.t3;
 --force-rmdir $MYSQLD_DATADIR1
 --remove_file $MYSQL_TMP_DIR/data57.zip
 
+--echo ###########################################################################
+--echo # PS#8050: PS 8.0 upgrade (from PS 5.7) crashes with Assertion failure: dict0dict.cc:1236:table2 == nullptr
+--echo ###########################################################################
+
+--copy_file $MYSQLTEST_VARDIR/std_data/upgrade/sys_table_duplicate_57.zip $MYSQL_TMP_DIR/data57.zip
+--file_exists $MYSQL_TMP_DIR/data57.zip
+--exec unzip -qo $MYSQL_TMP_DIR/data57.zip -d $MYSQL_TMP_DIR
+
+--let $restart_parameters = restart: --datadir=$MYSQLD_DATADIR1
+--let $wait_counter= 10000
+--source include/start_mysqld_no_echo.inc
+
+--echo ###########################################################################
```

Comment:
isn't there some user table in 5.7 user table to check content?

Reply:
I did not add any data in the user_table. just created one.

Reply:
Please add some user tables and some content. Just to know if the upgrade is really successful.

Reply:
added